web-ui/src/pages/AnniversaryReportPage.jsx

```diff
@@ -131,31 +92,13 @@ const BirthdayAnniversaryReportPage = () => {
           Run Search
         </Button>
       </div>
-      <div className="checkbox-row">
```

Comment:
We no longer need these checkboxes since this page only deals with anniversaries.